Assets/Scripts/PlayerController.cs

```diff
@@ -21,28 +41,31 @@ public class PlayerController : MonoBehaviour {
     [SerializeField]
     private ContactFilter2D groundContactFilter;
     [SerializeField]
-    private Text collectableText;
-    [SerializeField]
     private Slider overheatSlider;
 
-
     private Rigidbody2D rb2d;
     private Animator anim;
+    private AudioSource[] audioSources;
+    private AudioSource jumpSound;
+    private AudioSource deathSound;
     private Checkpoint currentCheckpoint;
     private Collider2D[] groundHitDetectionResults = new Collider2D[16];
     private bool grounded;
     private bool isDead = false;
     private bool isFacingRight = true;
-    private float collectables = 0f;
     private float overheat = 0f;
 
     private void Start()
     {
         rb2d = GetComponent<Rigidbody2D>();
         anim = GetComponent<Animator>();
+        audioSources = GetComponents<AudioSource>();
+
+        jumpSound = audioSources[0];
```

Comment:
I would make these of the type AudioClip and make them serialized fields and assign them in the editor.